helm/values.yaml

```diff
@@ -114,58 +117,66 @@ affinity: {}
 
 persistence:
   enabled: true
-  storageClassName: gp2
+  # storageClassName: "-"
   accessModes:
     - ReadWriteOnce
   size: 10Gi
   annotations: {}
 
 # Application
 public_env_variables:
-  UCENTRAL_ROOT: /ucentral-data
-  UCENTRAL_CONFIG: /ucentral-data
+  UCENTRALGW_ROOT: /ucentral-data
```

Comment:
Would also change it here to `/ucentralgw-data`. What do you think?

Reply:
Good point, will do

Reply:
Done